tests/TestingAppHost1/TestingAppHost1.ServiceDefaults/Extensions.cs

```diff
@@ -1,6 +1,3 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
You need to add this header back, it's causing a build failure.

Reply:
Oh man, I put it back and I'm gonna watch for the build to finish successfully this time 🤣.